domains/jiyathkhan.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Jiyath5516F",
+           "email": "",
+           "discord": "831836517516312586",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.QvqbHeShI7647_gtfKziEZkbLdpTS2jmcLwrj5OLVSWQjCYpEAEnLsmrHjXzA96i948IR23P5WsHraJIlvPIUL2hKNZRQx2xv7VaB7uBPR0NN4MyS5OJ7J22zqrV8Ahemb7_L6jhuEOmaqSas966JUgefyzvQl-wapBirjhFJDUOvPEtaVheI8ceNqCgzJQytNVSjSmSp0yJWCDf5SlswojVycHB8DDPx48J9ohOyKB_C8Wkl-U5vaceCB8cb8RBNeXuU_S_L9rnugNLIOGHtLacNq34PrD6aVfoTvLQr979hZXzLxXVBH_rgbTw28TEGM69LvCpNcrBvFVH_wjtcA.4V5C21i6ffviezfVCmv8jg.vojtJApVK6ARRzb9MjAxQ-_ezfP6yq1hUCN-EBDwNGGU3BWRS-ew5Js-dvK5eNSq7ecrup5XwO5De6tkGtozy5V0SZi0GfqZMc74fWM3xYo.TFqLmrPzaT6DqdJIOrXIDA"
+        },
+    
+        "record": {
+            "CNAME": "https://jiyath5516f.github.io/jiyath.is-a.dev"
```

Comment:
Invalid CNAME record. CNAME must be hostnames per DNS rules. This means not protocols, url paths, or parameters.